### DISCUSSION TITLE
问一个蠢问题

### DISCUSSION THREAD

**Discussion by owl342655596** [Gamuxorg/bbs#306] [建议和闲聊]
你们为什么一定要发打包的游戏呢？把原本的游戏安装文件发上来不行吗？

**Comment by wangling12** [comment]
尽可能添加依赖库，避免依赖问题，不然发行版A会说缺少liba,发行版B缺少libb，一个个回复起来很麻烦，另外还有可能依赖的库已经从源里面移除了，你所使用的发行版不一定能很容易的获取到。为了易用性，普通游戏玩家没必要，也不用掌握这些技能。
